projects/event-api.md

```diff
@@ -0,0 +1,90 @@
+
+Once created, please link to the project board here.
+
+Preemptively created a board with the relevant and related issues
+https://github.com/orgs/open-telemetry/projects/65
+
+### Deliverables
+
+Resolve the outstanding Event Issues / PR's which will define
+* What is an event? What does an event look like (what attributes identify it as an event) so that vendors, backends can perform any necessary routing, translation or storage.
+* How are events transported (Logs Only, Logs / Span Event, etc)
+* The semantic conventions to follow for defining "known" events
+  * Reserved (known) prefix's (either as the domain or name prefix) (possible known prefixes: `otel.`, `k8s.`, `azure.`, `aws.`, `ecs.`, `browser.`, `mobile.`, etc) all other "unknown" domains should/can be considered as "custom" by venders / backends.
+  * How / where are these are defined in semantic conventions (.yml)
+  * Updating tooling to properly handle the conventions
+  * Note: This group is not responsible for definition the semantic conventions of All domain specific events, only "how" those events will be represented / transported as an "event".
+* Define when an event should be used vs a `Span` or `LogRecord`
+
+We have been working on the definition already as part of the Client / RUM Sig and prototypes current exist in JavaScript in the web sandbox repo using `event.data` as the transport mechanism for the "Payload" of the event.
+
+* JS prototype of setting data https://github.com/open-telemetry/opentelemetry-sandbox-web-js/blob/auto-instrumentation-poc/pkgs/api-events/src/types/Event.ts#L34 
+* Swift setData() API https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetrySdk/Logs/LogRecordBuilderSdk.swift#L66 
+
+### Staffing / Help Wanted
+
+Who is currently planning to work on the project? If a project requires specialized domain expertise, please list it here. If a project is missing a critical mass of people in order to begin work, please clarify.
+
+#### Required staffing
+
+* Need: TC members (At least two sponsoring TC members.)
+* Need: Scheduled meeting where TC members and domain experts can attend
+* Need: more domain experts willing to define the OTel Event standard
+* Need: engineers willing to write prototypes in at least one more language other than JavaScript. Language should be fairly different from JavaScript.
+* Need: maintainers or approvers from those languages committed to reviewing the prototypes.
+
+@MSNev Nev Wylie - FTE project lead / browser domain expert
+@martinkuba Martin Kuba - Domain expert  and Maintainer / approver for JS
+@scheler Santosh Cheler - Domain expert
+@dennisme Matthew Dennison - Golang sdk
+@patrickhousley Patrick Housley - Event domain expert
+@breedx-splk Jason Plumb - Event domain expert
```

Comment:
Let's organize the project contributors into groups like the [other projects](https://github.com/open-telemetry/community/tree/main/projects): Project leads, Sponsoring TC members, Contributing Engineers, Implementation Maintainers or Approvers. 

Also, we discussed in the 9/27 TC meeting and @tigrannajaryan and I agreed to be TC sponsors. @tedsuo from the GC mentioned that he would participate, and @jmacd will help review PRs to the specification. Not sure how @tedsuo and @jmacd fit into the staffing categories in this document. Maybe they can comment?